src/scss/components/_list.scss

```diff
@@ -99,9 +105,8 @@
       float: left;
     }
 
-    &::before {
-      content: '';
-      margin-right: 5px;
+    &:last-child {
+      margin-bottom: 0;
```

Comment:
Apply this, does have a side effect in some scenario?

Reply:
I see the others list types and does not have side effect.